### DISCUSSION TITLE
More options in Navigation

### DISCUSSION THREAD

**Discussion by marco-santulli** [Budibase/budibase#3836] [Ideas]
Hi,

If I get it right, custom CSS really wipes out the whole CSS of the "top" element one modifies.  There is a fallback using spectrum on the bits where it is available.   I guess making custom css apply on nested elements is complicated, but then maybe we can get a fix on the menu, because right now it looks like this:

![image](https://user-images.githubusercontent.com/82715690/146695221-163ce2e3-2aaa-4924-b710-e89aa4cf5a74.png)

but with a simple (maybe) we could get it all on a single line improving real-eastate?
flex-direction: row;

![image](https://user-images.githubusercontent.com/82715690/146695282-45b61e2c-9af6-49e1-b8fe-d4d2767074c2.png)

**Comment by mjashanks** [comment]
I see what you mean. You can of course choose `Navigation: None` and build out the navigation yourself, with containers. It's a bit of a pain, but it works.

**Reply by marco-santulli** [reply]
Hi Mike,

sounds like a good idea

thx

Marco

**Reply by agaheu** [reply]
Hi All,
Is there any example or definiton for custom menu container?
